slides/slides.md

```diff
@@ -550,3 +550,77 @@ guard against these.
 ::: notes
 
 :::
+
+### Part 3.1
+
+<h3>Code reviews for OSS: the case in Anaconda</h3>
+
+### A few words about Anaconda
+
+- Anaconda (prev. _Continuum Analytics_) is well-known in the Data Science community for its contribution to open source projects and the PyData ecosystem
+
+. . .
+
+- As a company, Anaconda is a  _RemoteFirst_ organisation
+
+	+ fully-distributed teams set to work asynchronously (w/ timely sync catch ups)
+
+::: notes
+
+Anaconda sustains NumFOCUS, which is the non-profit umbrella org. supporting the main projects in the Python scientific ecosystem
+
+:::
+
+### Code Review in Anaconda
+
+(_Asynch_) **Code Review** plays a huge role in Anaconda
+
+. . . 
+
+- Code review is a standard practice in all projects used in the majority of cases to:
+	+ Guarantee Software compliance (_audit_)
+	+ Enforce Team collaboration
+	+ Provide a welcoming and friendly  _onboarding_ for newcomers 	
+	
+. . .
+
+- Code Review sets and (easily) adapts differently depending on specific Projects requirements
+
+. . . 
+
+- The type of the team is very important, for example:
+	+ All IT-experts; IT people w/ diff experience or backgrounds; IT & non-IT people.. 
```

Comment:
I guess what is particularly interesting in our case is HOW doe Anaconda (as an organisation) achieves (enforce?) such levels of adaptation.